otx/mpa/modules/models/heads/custom_atss_head.py

```diff
@@ -168,18 +168,16 @@ def loss_single(
             pos_centerness = centerness[pos_inds]
 
             centerness_targets = self.centerness_target(pos_anchors, pos_bbox_targets)
-            pos_decode_bbox_pred = self.bbox_coder.decode(pos_anchors, pos_bbox_pred)
-            pos_decode_bbox_targets = self.bbox_coder.decode(pos_anchors, pos_bbox_targets)
+            if self.reg_decoded_bbox:
```

Comment:
```suggestion
            if not self.reg_decoded_bbox:
```
Isn't the contrary right?

Reply:
I don't think so. Targets are encoded when self.reg_decoded_bbox is False. Therefore if self.reg_decoded_bbox is False, then the bbox_pred don't need to decode. However if self.reg_decoded_bbox is True, then the bbox_pred should be decoded since the bbox_targets are not encoded